src/reducers/reducerTypes.ts

```diff
@@ -0,0 +1,9 @@
+export interface IRecipe {
```

Comment:
Generally I've seen interfaces prefaced with an `I` before them - just a code style I prefer, we definitely do not need to follow it!